soccer/src/soccer/params.hpp

```diff
@@ -0,0 +1,7 @@
+#pragma once
+
+#include <rj_param_utils/param.hpp>
+
+constexpr auto kSoccerParamModule = "soccer";
```

Comment:
Nit: change this param module to `kGlobalParamModule`. We'll probably eventually be using this module in #1586.